frontend/src/components/StateCircle/stateCircleStyle.ts

```diff
@@ -18,4 +18,4 @@ export const circleSelectedClass = css`
 export const textStyles = {
   userSelect: 'none',
   fill: 'var(--stroke)'
-}
+} as const
```

Comment:
Was getting type errors so it needed to be extra const so that the strings were restricted properly